addons/sales_team/models/crm_team.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from openerp import api, fields, models
+
+
+class CrmTeam(models.Model):
+    _name = "crm.team"
+    _inherit = ['mail.thread', 'ir.needaction_mixin']
+    _description = "Sales Team"
+    _order = "name"
+
+    @api.model
+    def _get_default_team_id(self, user_id=None):
```

Comment:
Actually I think you have to use a return decorator. The method should return an id in the old API, but a record set in the new API, if I am not wrong.